src/ActionParser.h

```diff
@@ -16,16 +16,19 @@
 #include "RelationParser.h"
 #include "RegexMatch.h"
 
+// Similiar to SYMBOL_IDENTIFIER, except `(` and `)` are allowed
+#define ACTION_SYMBOL_IDENTIFIER "(.+)"
```

Comment:
Yes, you are right. Updated.

I mis-interpreted the `[` / `]` in the previous regex as they are also regex characters.